clusterctl/examples/vsphere/generate-yaml.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+set -e
+
+PROVIDERCOMPONENT_TEMPLATE_FILE=provider-components.yaml.template
+PROVIDERCOMPONENT_GENERATED_FILE=provider-components.yaml
+
+MACHINE_CONTROLLER_SSH_PUBLIC_FILE=vsphere_tmp.pub
+MACHINE_CONTROLLER_SSH_PUBLIC=
```

Comment:
Is this supposed to be empty?

Reply:
Yep. It is the contents of the file. Will be populated after generation.